src/components/events/Event/style.scss

```diff
@@ -118,16 +120,9 @@ $ehw-animation-speed: 200ms;
 
 .Event__join-event {
   position: absolute;
-  color: $ehw-grey-light;
-  left: 30%;
-  font-size: 1em;
-  font-weight: normal;
-  line-height: 2.5;
+  left: 50%;
   top: 110%;
-  width: 40%;
-  text-decoration: none;
   transition-delay: 200ms;
-  background-color: $ehw-green;
-  border-radius: 3px;
-  box-shadow: -1px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  margin: 0;
+  transform: translateX(-50%) translateY(-50%);
```

Comment:
Pretty confident we've dropped / are very soon dropping support for those browsers.